src/components/plots/stackedAreaPlot.ts

```diff
@@ -69,6 +69,13 @@ export module Plots {
       return this;
     }
 
+    protected _updateExtentsForAttr(attr: string) {
+      super._updateExtentsForAttr(attr);
+      if ((attr === "x" || attr === "y") && this._projectorsReady()) {
+        this._updateStackExtents();
+      }
```

Comment:
`Plots.Stacked` should from my gathering be some sort of useful utility.  What we're running into from my view is that `Plots.StackedBar`/`Plots.StackedArea` need to extend from their respective parents but needs to utilize some object or thing in order to figure out offsets.